core/modules/user/user.admin.inc

```diff
@@ -552,7 +562,8 @@ function user_admin_roles($form, $form_state) {
         ),
       );
     }
-    if ($role_name !== BACKDROP_ANONYMOUS_ROLE && $role_name !== BACKDROP_AUTHENTICATED_ROLE) {
```

Comment:
looks like there's an accidental change here. This is the kind of thing that could introduce errors, so it's best not to change anything other than what the issue says you're changing.

Reply:
I was trying to be consistent with https://github.com/backdrop/backdrop/pull/3203/files#diff-2b236f6288c8e14c97892a1e54258197R179 ...happy to revert.